packages/integration-tests/scripts/bridgeERC20.ts

```diff
@@ -31,7 +32,7 @@ const TOKEN_AMOUNT = '10';
   // on the Ethereum chain for interacting with the Fuel chain
   console.log('Setting up environment...');
   const env: TestEnvironment = await setupEnvironment({});
-  const ethAcct = env.eth.signers[1];
+  const ethAcct = env.eth.signers[0];
```

Comment:
What's the reason for changing the signer?

Reply:
I'm not sure

cc @DefiCake

Reply:
The block committer is using signer[1]. This is one of the couple of reasons the tests were showing flakiness, because if the wallet trying to transact with the bridges and the block committer send transactions close to each other, there will be nonce reusability issues